docs/docsite/rst/community/release_managers.rst

```diff
@@ -1,7 +1,7 @@
 .. _release_managers:
 
 **************************
-Release manager guidelines
+Release Manager Guidelines
```

Comment:
Thanks for catching that. I sometimes still use Very Formal Capitalized Titles. It's my inner Victorian novelist coming out.

Reply:
😆